src/modules/ruby.rs

```diff
@@ -0,0 +1,60 @@
+use ansi_term::Color;
+use std::process::Command;
+
+use super::{Context, Module};
+
+/// Creates a module with the current Ruby version
+///
+/// Will display the Ruby version if any of the following criteria are met:
+///     - Current directory contains a `.rb` file
+///     - Current directory contains a `Gemfile` file
+///     - Current directory contains a `.ruby-version` file
```

Comment:
I would drop support here for `.ruby-version` until we specifically support RVM.
```suggestion
```

Reply:
Done! :smile: